Source/WebInspectorUI/UserInterface/Views/FontDetailsPanel.js

```diff
@@ -35,18 +33,23 @@ WI.FontDetailsPanel = class FontDetailsPanel extends WI.StyleDetailsPanel
         this._fontStyles = null;
         this._fontVariationRowsMap = new Map;
         this._basicPropertyRowsMap = new Map;
+        this._fontPropertyRowsMap = new Map;
         this._basicPropertyNames = ["font-size", "font-style", "font-weight", "font-stretch"];
 
         this._abortController = new AbortController;
-        this._throttledUpdate = new Throttler(() => { this.update() }, 100);
-        this._debouncedUpdate = new Debouncer(() => { this._skipNextUpdate = false; this.update(); });
+        this._throttledUpdate = new Throttler(() => {
```

Comment:
ditto above

i realize that this doesn't match the expected style of Web Inspector and can understand wanting to fix it, but again we dont make drive-by style changes unless it's related to the overall purpose of the PR